vyper/semantics/types/function.py

```diff
@@ -336,6 +336,10 @@ def from_FunctionDef(
         # return types
```

Comment:
I kind of want to refactor this function so all the logic for `__init__` and `__default__` go in their own blocks, but this is fine for now.